### PR TITLE
Fix 380 for 1.2.2

### PR DESCRIPTION
**Pull request by marcmengel**
Update condor_submit and condor_submit_dag wrappers to pass actual arg-parser args to get_schedd(). 
Also add a test of actually submitting with condor_submit.

### CODE REVIEW

File: lib/get_parser.py [hunk 1]
```diff
@@ -141,6 +141,12 @@ def get_base_parser(add_condor_epilog: bool = False) -> argparse.ArgumentParser:
         action="store_true",
         help="Use jobsubdevgpvm01 etc. to submit",
     )
+    parser.add_argument(
```
Comment (shreyb) [inline lib/get_parser.py:144]: Should this actually be here though?  A user, for example, can't run `jobsub_q -G dune --job-info 12345@jobsub01.fnal.gov`, right?

**Review by shreyb — CHANGES_REQUESTED** [review]
Just the 1 thing in get_parser.

File: lib/get_parser.py (hunk 1 above)
Comment (marcmengel) [reply lib/get_parser.py:144]: Hm.... well, we could add a third layer; one common to condor_submit, condor_submit_dag and jobsub_submit,
and put the --job-info flag in there.  On the other hand, it is  vacuously correct as is, if jobsub_q **does** call the job-info script for all of the (zero) jobs it suubmits...

File: lib/get_parser.py [hunk 2]
```diff
@@ -156,6 +156,17 @@ def get_base_parser(add_condor_epilog: bool = False) -> argparse.ArgumentParser:
     return parser
 
 
+def get_submit_parser(add_condor_epilog: bool = False) -> argparse.ArgumentParser:
```
Comment (shreyb) [inline lib/get_parser.py:159]: Could you please add a quick docstring here explaining that this is the common parser used by all submission binaries (condor_submit, jobsub_submit)?

**Review by shreyb — CHANGES_REQUESTED** [review]
Looks great.  Just the one request for a docstring, and I think this is good to go.

File: lib/get_parser.py (hunk 1 above)
Comment (shreyb) [reply lib/get_parser.py]: I should have read the file changes before writing that comment.  Thanks for putting in the extra layer of parsers.

**Pull request merged**